data/competencies.yml

```diff
@@ -925,3 +925,15 @@
   level: senior1-to-senior2
   area: leadership
   domain: null
+
+
+- id: f173b7e9-1c46-4afd-a659-d264aa173759
+  summary: Contributes to inclusive organisational culture
+  examples:
```

Comment:
I think it might be worth repeating some of the examples from S1, because they're important and it's good to have them in one place, if you're looking at a specific level

Reply:
We try not to repeat examples as it's easy for changes like to to get out of sync. [Engineers are expected to continue meeting competencies from earlier levels](https://engineering-progression.ft.com/docs/faq/#do-i-need-to-meet-competencies-from-earlier-levels), which reduces the need for duplication. Are any of the examples from the S1 competency possible to _expand_ on in S2?